src/generic/grid.cpp

```diff
@@ -3859,11 +3862,75 @@ void wxGrid::PrepareDCFor(wxDC &dc, wxGridWindow *gridWindow)
     dc.SetDeviceOrigin(dcOrigin.x, dcOrigin.y);
 }
 
+void CheckDoDragScroll(wxGrid *grid, wxGridSubwindow *eventGridWindow,
+    wxGridSubwindow *gridWindow, wxPoint event_pos,
+    bool horizontal, bool vertical)
+{
+    // scroll when at the edges or outside the window
+    // eventGridWindow: the window that received the mouse event
+    // gridWindow: the same or the corresponding non-frozen window
+    int w, h;
+    eventGridWindow->GetSize(&w, &h);
+    // ViewStart is scroll position in scroll units
+    wxPoint sxy = grid->GetViewStart();
```

Comment:
Minor again, but `sxy` is a rather unreadable name. Let's call it `ptStart` or just `origin` perhaps. And let's please also make it const to indicate that it's not going change:
```suggestion
    const wxPoint origin = grid->GetViewStart();
```